src/features/cell-renderers/selectableRowRenderer.js

```diff
@@ -0,0 +1,17 @@
+// @flow
```

Comment:
We should have a different folder for this. This is a rowRenderer and not a cellRenderer